### PR TITLE
Test tutorials on travis

### PR DESCRIPTION
**Pull request by fritzo**
This adds support for testing some of our tutorials on travis.

## Why?

We're changing Pyro and we want to avoid bit rot of tutorials.

## How?

This uses a kludgey mechanism of defining a boolean `smoke_test = 'CI' in os.environ` in each tutorial. This is a little ugly, so we might something cleaner before Pyro 0.2 release, e.g. replacing all strings of the form `num_iters = [0-9]*` with the string `num_iters = 1`.

### CODE REVIEW

**Comment by fritzo** [comment]
@neerajprad looks like build is failing due to wheel issues

**Comment by neerajprad** [comment]
@fritzo - I had updated the wheel files for Multinomial, but forgot to overrride the permissions to public. Restarted the tests, it should work now.

File: tutorial/source/gmm.ipynb [hunk 1]
```diff
@@ -29,7 +30,8 @@
     "from pyro.optim import Adam\n",
     "from pyro.infer.svi import SVI\n",
     "from pyro.distributions import categorical, normal\n",
-    "softmax = Softmax(dim=-1)"
+    "softmax = Softmax(dim=-1)\n",
+    "smoke_test = ('CI' in os.environ)"
```
Comment (neerajprad) [inline tutorial/source/gmm.ipynb]: Were the changes in this file due to just running the `nbval` command?

**Comment by null-a** [comment]
This seems reasonable to me. Once #680 is merged the AIR tutorial should be working again and could be added to these smoke tests. I could do this as a follow-up PR if you like.

**Comment by fritzo** [comment]
@neerajprad sorry it took me so long to fix this PR. We'll need this before **The Big Refactoring** next week.

**Pull request merged**